tests/test_api.py

```
@@ -0,0 +1,253 @@
+
+def test_wiki_get_or_404(app):
+    """Test get_or_404 for existing and missing pages."""
+    with app.test_request_context():
+        wiki = WikiBase(app.config["WIKI_CONTENT_DIR"])
+
+        # Existing page
+        page = wiki.get_or_404("home")
+        assert page.title == "Home"
+
+        # Missing page
+        with pytest.raises(Exception):
```

Ruff (0.15.7)

[warning] Do not assert blind exception: `Exception`

(B017)